src/ENAIngest.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+import os
+from helperFunctions import getValueExamples
+
+output = "ena_assembly_all_fields.tsv"
+dataOutLoc = "../data"
+generatedOutLoc = "../generatedFiles"
+
+outPath = os.path.join(dataOutLoc, output)
+if not os.path.exists(outPath):
```

Comment:
We probably want to download a new copy every time this is run, so we get most recent data... so wondering if we include a flag or days since data was last downloaded and force it to download new copy? Just something to think about for next commit...